src/pages/AllPosts/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import AppFilterMenu from '../../components/AppFilterMenu';
+import PostResults from '../../components/PostResults';
+import './style.css';
+import { Grid } from '@material-ui/core';
+
+export default function Index() {
+  return (
+    <Grid container justify="center" alignItems="center">
```

Comment:
See here that the grid is already centred so adding extraneous styling is pointless since this is the parent in the parent:child relationship of flexbox